core/tests/protractor_desktop/communityDashboard.js

```diff
@@ -43,3 +45,59 @@ describe('Community dashboard page', function() {
     general.checkForConsoleErrors([]);
   });
 });
+
+describe('Admin page community reviewer form', function() {
+  var adminPage = null;
+  var communityDashboardPage = null;
+
+  beforeAll(function() {
+    adminPage = new AdminPage.AdminPage();
+    communityDashboardPage = (
+      new CommunityDashboardPage.CommunityDashboardPage());
+    users.createUser('translator@community.com', 'translator');
+    users.createUser('voiceartist@community.com', 'voiceartist');
+    users.createUser('questionreviewer@community.com', 'questionreviewer');
+    users.createAdmin('management@adminTab.com', 'management');
+  });
+
+  beforeEach(function() {
+    users.login('management@adminTab.com', true);
+  });
+
+  it('should allow admin to add translation reviewer', function() {
+    adminPage.get();
+    adminPage.assignTranslationReviewer('Hindi', 'translator');
+    users.logout();
+
+    users.login('translator@community.com');
+    communityDashboardPage.get();
+    communityDashboardPage.expectUserToBeTranslationReviewer('Hindi');
```

Comment:
The language name gets reused a lot in this test, so definitely consider making it a constant variable